files/scripts/provision.sh

```diff
@@ -14,9 +14,11 @@ fi
 
 /opt/muna/scripts/setup_ssenv "${NAMESPACE}"
 /opt/muna/scripts/setup_secrets "${NAMESPACE}"
-/opt/muna/scripts/setup_ssl "${NAMESPACE}"
-
 systemctl reload apache2
 
-# Only try to reload nginx if it is installed and running
-(systemctl is-active --quiet nginx && nginx -t && systemctl reload nginx) || true
\ No newline at end of file
+# Only try to setup SSL and reload nginx if it is installed and running
+if [ systemctl is-active --quiet nginx ]; then
```

Comment:
For this to work you'd need to wrap the command with a `$(...)` (this is currently invalid bash). Also, the success return value is "0" so the following block will only currently execute if nginx is NOT active (probably best to do explicit comparison).

Also, maybe good to leave that `nginx -t` in there, beause that is probably meant to produce debug info into deploy log, which is handy ("reload" will just say check your journal).

Overall, maybe best to stick with the original :)

Reply:
Have fixed this up (forgot to update command after testing locally). I have also added back in `nginx -t` for debugging output as requested.

The change here was intended to only run `setup_ssl` if Nginx is installed as currently this is triggered on CWP Instances and throws a non-blocking error.